The error are clear, for example:

```
[ 41%] Building CXX object drivers/telescope/CMakeFiles/indi_lx200generic.dir/lx200_esp32go.cpp.o
/__w/indi/indi/indi/drivers/telescope/lx200_esp32go.cpp: In member function ‘virtual bool LX200_esp32go::SetTrackMode(uint8_t)’:
/__w/indi/indi/indi/drivers/telescope/lx200_esp32go.cpp:403:42: error: unused parameter ‘mode’ [-Werror=unused-parameter]
  403 | bool LX200_esp32go::SetTrackMode(uint8_t mode)
      |                                  ~~~~~~~~^~~~
[ 41%] Linking CXX executable indi_simulator_ccd
```

So why isn't mode used? If on purpose, you can use INDI_UNUSED(mode); to ignore.

--- a/drivers/telescope/lx200_esp32go.cpp
+++ b/drivers/telescope/lx200_esp32go.cpp
@@ -404,7 +404,9 @@ bool LX200_esp32go::SetTrackMode(uint8_t mode)
 {
     if (isSimulation())
         return true;
-
+    
+    LOGF_DEBUG("SetTrackMode: %d", mode);
+    
     // FW not ready yet
 /*
     LOG_INFO("SetTrackMode");

--- a/drivers/telescope/lx200_esp32go.h
+++ b/drivers/telescope/lx200_esp32go.h
@@ -44,15 +44,15 @@ class LX200_esp32go : public LX200Generic
         LX200_esp32go();
         ~LX200_esp32go() {}
 
-        const char *getDefaultName();
-        bool initProperties();
-        bool updateProperties();
-        void ISGetProperties(const char *dev);
+        const char *getDefaultName() override;
+        bool initProperties() override;
+        bool updateProperties() override;
+        void ISGetProperties(const char *dev) override;
         //virtual bool ISNewSwitch(const char *dev, const char *name, ISState *states, char *names[], int n); //override;
-        virtual bool ISNewNumber(const char *dev, const char *name, double values[], char *names[], int n);
+        virtual bool ISNewNumber(const char *dev, const char *name, double values[], char *names[], int n) override;
 
     protected:
-        virtual bool UnPark();
+        virtual bool UnPark() override;
 
         
         virtual bool ReadScopeStatus() override;
@@ -85,9 +85,9 @@ class LX200_esp32go : public LX200Generic
         void splitString(const char *str, char array[][100], int *count);
 
         // ------------ FocuserInterface
-        IPState MoveAbsFocuser (uint32_t targetTicks);// override;
-        IPState MoveRelFocuser (FocusDirection dir, uint32_t ticks);// override;
-        bool AbortFocuser ();// override;
+        IPState MoveAbsFocuser (uint32_t targetTicks) override;
+        IPState MoveRelFocuser (FocusDirection dir, uint32_t ticks) override;
+        bool AbortFocuser () override;
         int UpdateFocuser(); //Return = 0 good, -1 = Communication error
 
 };

